scrape_rec/db_wrapper.py

```diff
@@ -0,0 +1,59 @@
+import sqlalchemy as sa
+
+from os import environ
+from sqlalchemy.ext.declarative import declarative_base
+
+
+POSTGRES_USER = 'postgres'
+POSTGRES_PASSWORD = environ.get('POSTGRES_PASSWORD')
+POSTGRES_HOST = 'localhost'
+POSTGRES_PORT = 5342
+POSTGRES_DB = 'realestate'
+
+
+db_string = 'postgres://{user}:{password}@{host}:{port}/{db}'.format(
+  user=POSTGRES_USER,
+  password=POSTGRES_PASSWORD,
+  host=POSTGRES_HOST,
+  port=POSTGRES_PORT,
+  db=POSTGRES_DB
+)
+
+db = sa.create_engine(db_string)
+base = declarative_base()
+
+
+class RealestateApartment(base):
+    __tablename__ = POSTGRES_DB
+
+    fingerprint = sa.Column(sa.String, primary_key=True)
+
+    # Mandatory
+    title = sa.Column(sa.String, nullable=False)
+    description = sa.Column(sa.String, nullable=False)
+    scraped_date = sa.Column(sa.DateTime, nullable=False)
+    price = sa.Column(sa.Integer, nullable=False)
+    currency = sa.Column(sa.String, nullable=False)
```

Comment:
Does this default to euro? It feels like it should default to euro as it's usually the case

Reply:
I wouldn't default, I'd rather drop the item.